src/app/modules/home/pages/home.view.html

```diff
@@ -1,89 +1,85 @@
 <div class="container">
-<section>
-       
-<ngb-carousel>
-        
-        <ng-template ngbSlide>
+  <section>
+         
+  <ngb-carousel>
+          
+          <ng-template ngbSlide>
+                  <ng-template ngbSlide>
+                          <div class="picsum-img-wrapper">
+                            <img style="border-radius: 25px;" [src]="images[0]" alt="Random third slide" height="400" width="1060">
```

Comment:
no inline css allowed. pls fix this  for img ,h2 tag . fix for all the tags u feel is duplicated.